discovery/spec.md

```diff
@@ -588,10 +588,17 @@ The result MUST be a JSON object of the following form:
 ```
 {
   "servicefilterattributes": [ "name", ... ],
-  "pagination": "true"
+  "pagination": true
 }
 ```
 
+The following additional constraints apply:
+- The list of feature names are case sensitive.
```

Comment:
are => is? Or rewrite.

Reply:
"fearures" -> "features" on 585

Reply:
went with "the feature names are case sensitive"